packages/xchain-bitcoin/src/client.ts

```diff
@@ -198,10 +192,13 @@ class Client implements BitcoinClient, XChainClient {
    * @throws {"Phrase must be provided"} Thrown if phrase has not been set before.
    * @throws {"Address not defined"} Thrown if failed creating account from phrase.
    */
-  getAddress = (): Address => {
-    if (this.phrase) {
+  getAddress = (index = 0): Address => {
+    if (index < 0) {
+      throw new Error('index must be greater than zero')
+    }
+    if (this.phrase !== '') {
```

Comment:
Why a change from
```
if (this.phrase)
```
to
```
(this.phrase !== '')
```

Reply:
reverted. not sure why i changed it at this point.